### DISCUSSION TITLE
Welcome to estate Discussions!

### DISCUSSION THREAD

**Discussion by Vodhin** [Vodhin/estate#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kanonimpresor** [comment]
Wow! The work he has done is incredible!

I was immediately cloned, after seeing this marvel. It is translated into Spanish and Portuguese and I will soon start working.

My name is Martin, I live in Portugal and was born in Argentina, and looking through my antiques I find myself with e107 records that I started using in 2010 (after the tube system has to be rewritten)

Today I rediscovered this fantastic CMS, so I'm updating the information to start creating themes again.

I will contact you to apply your plugin.

a hug!

**Reply by Jimmi08** [reply]
e107 is awesome, just people don't know about it.  If you are able to change little things, there is nothing better.

**Reply by Vodhin** [reply]
> Wow! The work he has done is incredible!
> 
> I was immediately cloned, after seeing this marvel. It is translated into Spanish and Portuguese and I will soon start working.
> 
> My name is Martin, I live in Portugal and was born in Argentina, and looking through my antiques I find myself with e107 records that I started using in 2010 (after the tube system has to be rewritten)
> 
> Today I rediscovered this fantastic CMS, so I'm updating the information to start creating themes again.
> 
> I will contact you to apply your plugin.
> 
> a hug!

Hello Martin! Thank you for the kind words. Let me know when you have finished language files. I think you can simply attach them to a post in this thread. I can then add them to the repository.
